### PR TITLE
Add CloudWatch metrics for DelayedJob

### PR DESCRIPTION
**Pull request by iant**
```
aws cloudwatch put-metric-data --metric-name MemoryUsage --namespace DelayedJob --value 12 --unit Count --region=eu-west-2 --dimensions InstanceId=$(ec2-metadata --instance-id | cut -d " " -f 2)
```

### CODE REVIEW

File: .ebextensions/cloudwatch.config [hunk 1]
```diff
@@ -11,12 +11,16 @@ sources:
...
 container_commands:
-  01-setupcron:
+  01-setupmetricscron:
     command: |
       echo '*/5 * * * * root perl /opt/cloudwatch/aws-scripts-mon/mon-put-instance-data.pl `{"Fn::GetOptionSetting" : { "OptionName" : "CloudWatchMetrics", "DefaultValue" : "--mem-util" }}` >> /dev/null 2>&1' > /etc/cron.d/cwpump
-  02-changeperm:
+  02-setupmemoryusagemetricscron:
+      echo '*/5 * * * * aws cloudwatch put-metric-data --metric-name MemoryUsage --namespace DelayedJob --value=$(./current/script/delayed_job_memory_usage.sh) --unit Count --region=eu-west-2 --dimensions InstanceId=$(ec2-metadata --instance-id | cut -d " " -f 2)' > /etc/cron.d/cwpump
+  03-setupnumberofprocessesscron:
+      echo '*/5 * * * * aws cloudwatch put-metric-data --metric-name Processes --namespace DelayedJob --value=$(./current/script/delayed_job_number_of_processes.sh) --unit Count --region=eu-west-2 --dimensions InstanceId=$(ec2-metadata --instance-id | cut -d " " -f 2)' > /etc/cron.d/cwpump
```
Comment (iant) [inline .ebextensions/cloudwatch.config:20]: set for every 5 mins as above

Comment (iant) [reply .ebextensions/cloudwatch.config:20]: @ldodds i'm thinking we should remove these from this pr and set up manually to test.  what do you think?

Comment (ldodds) [reply .ebextensions/cloudwatch.config:20]: As discussed, remove this from this PR and create a separate PR with a change to the cronjob.config to create the necessary cron file.

**Review by ldodds — CHANGES_REQUESTED** [review]

File: .ebextensions/cloudwatch.config (hunk 1 above)
Comment (iant) [reply .ebextensions/cloudwatch.config]: :+1: done @ldodds

**Review by ldodds — APPROVED** [review]

**Pull request merged**